src/loader.ts

```diff
@@ -0,0 +1,44 @@
+import { Client, Collection } from "discord.js";
+import { Command } from "./Command";
+import * as fs from "fs";
+import { ButtonHandler } from "./ButtonHandler";
+
+export async function loadCommands(): Promise<Collection<string, Command>> {
+	const loadedCommands = new Collection<string, Command>();
+	await Promise.all(
+		fs
+			.readdirSync("./dist/commands")
+			.filter((f) => f.endsWith(".js"))
```

Comment:
not including all the files that start with `sample` to have sample files for the commands/buttons/events showing how to implement new commands/buttons/events with the right fromat
```suggestion
			.filter((f) => f.endsWith(".js") && !f.startsWith(sample))
```

Reply:
just put .disabled at the end like everyone

Reply:
okay